src/app/pages/page-not-found/page-not-found.component.html

```diff
@@ -0,0 +1,13 @@
+<app-hill-background cover></app-hill-background>
+<app-back-button top-left routerLink="/"></app-back-button>
+<div class="centered-container">
+  <div>
+    <mat-card>
+      <mat-card-title class="title">Whoops</mat-card-title>
```

Comment:
```suggestion
      <mat-card-title>Page Not Found</mat-card-title>
```

I prefer more informative wording here.